java/arcs/core/data/proto/AccessPathProtoDecoder.kt

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I've been putting these kinds of extension functions inside files name FooProto.kt, but I see elsewhere we've been calling them FooProtoDecoder.kt / FooProtoEncoder.kt. 

It'd be nice to settle on one. Personally I don't see any reason to separate out the encoding and decoding into separate files, WDYT?

Reply:
We don't have any encoders for the manifest at this point. In that sense, this directory is consistent. 

I will be OK to name them as FooProto.kt, but we could defer this until we add encoders?

Reply:
It's fine to leave it as-is for now. I'll probably come through later and rename these files to all be consistent with each other.